README.md

```diff
@@ -27,43 +36,58 @@ You can also [download the latest release][hugo-release] and checkout the
 [hugo-release]: https://github.com/spf13/hugo/releases "Download Latest Hugo Release"
 [hugo-quick-guide]: http://gohugo.io/overview/quickstart/ "Hugo Quickstart Guide"
 
-## Installing Gulp
+### Installing Gulp
 
 In order to build the assets for the microsite, you will need to install the
-`gulp` command-line tool. Because Gulp depends on `npm` and `node` during
-runtime, please make sure you have them installed on your system. Instructions
-for installing `node`, including `npm`, [can be found here] [node-install].
-
-[node-install]: https://nodejs.org/en/download/ "NodeJS Downloads"
+`gulp` command-line tool.
 
 ```sh
 npm install --global gulp-cli
 ```
 
-Once `gulp` is installed locally, navigate to this directory in your Terminal
+### Installing Scss Lint
+
+The Sass file linter uses the Ruby gem `scss_lint`. This gem must be installed
+on your machine in order to run any tasks that depend on the `scss-lint` binary.
+
+```sh
+gem install scss_lint
+```
+
+### Development
+
+Once `gulp` is installed globally, navigate to this directory in your Terminal
 and run `npm install` to bring in the asset-pipeline's dependencies.
 
 Once that is complete, run `gulp` in your Terminal to get a list of tasks.
 
 ```sh
-gulp
-
-   Using gulpfile ~/Developer/ffd-microsite/gulpfile.js
-   Starting 'default'...
-   v1.0.0 ffd-microsite
-    ______ ______ _____
-   /\  ___/\  ___/\  __-.
-   \ \  __\ \  __\ \ \/\ \
-    \ \_\  \ \_\  \ \____-
-     \/_/   \/_/   \/____/
-
-   Available tasks
-   $ gulp watch
-   Watch for changes and build the asset-pipeline
-   $ gulp [ production, no-test ] build
-   Build the asset-pipeline with optional production and no-test flags
-   $ gulp clean-all
-   Removes files and directories generated by the asset-pipeline
-   $ gulp [ production ] website
-   Runs the gulp watch and hugo serve
+❯❯❯❯❯❯❯ gulp
```

Comment:
Small change. Either change to `$ gulp` for consistency or remove the `>>>>>` entirely.